apps/privacy-panel/locales/privacypanel.en-US.properties

```diff
@@ -88,24 +94,26 @@ passphrase-invalid = You cannot use special characters!
 passphrase-different = Confirmation must match passphrase!
 pin-empty = Passcode lock is empty!
 pin-invalid = Wrong Passcode lock!
+passcode-long = Passcode should be 4 digits!
 pin-different = Wrong Passcode lock!
-sim-invalid = Wrong SIM PIN!
+sim-invalid = Wrong SIM PIN! {{ value }} tries left!
```

Comment:
I think we might need to change the key here? It's how the localization system knows there's a change to update translations. Otherwise other locales might be missing the new content.